pom.xml

```diff
@@ -72,14 +72,14 @@
     </dependency>
     <!-- api client -->
     <dependency>
-      <groupId>com.github.genome-nexus.genome-nexus-java-api-client</groupId>
+      <groupId>com.github.averyniceday.genome-nexus-java-api-client</groupId>
```

Comment:
Reminder here to and [here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline/pull/119/files?diff=unified&w=1#diff-600376dffeb79835ede4a0b285078036R77) to update the `artifactId` before merging.